### ISSUE TITLE
preprocessing for cam

### ISSUE THREAD

**Issue by aguang1201** [brucechou1983/CheXNet-Keras#38]
When you do cam preprocessing,the function transform_batch_images is not called.But the result of cam images is right.Why?Did you trained model without the function transform_batch_images?

**Comment by dimitree54** [comment]
I also noticed this difference. There is no normalization (with imagenet_mean and imagenet_std values) in cam.py because transform_batch_images is not called. Because of it test.py and cam.py return different predictions for same images. As I understand this normalization is a part of input pipeline during training, so  this function should be called in cam.py, too.

**Comment by artembelopolsky** [comment]
yes, and the results look quite different when you add the normalization.
